clang/include/clang/Sema/Sema.h

```diff
@@ -12304,7 +12306,14 @@ class Sema final {
   /// Checks if Callee function is a device function and emits
   /// diagnostics if it is known that it is a device function, adds this
   /// function to the DeviceCallGraph otherwise.
-  void checkSYCLDeviceFunction(SourceLocation Loc, FunctionDecl *Callee);
+  ///
+  /// \return false if Callee is not a device function
```

Comment:
Could you please rewrite this comment so it will match with reality?
Something like comment for `CheckCUDACall`:
```
/// Check whether we're allowed to call Callee from the current context.
///
/// - If the call is never allowed in a semantically-correct program
///  emits an error and returns false.
///
/// - If the call is allowed in semantically-correct programs, but only if
///   it's never codegen'ed, creates a deferred diagnostic to
///   be emitted if and when the caller is codegen'ed, and returns true.
///
```